features/fetch-crl/config.pan

```diff
@@ -16,25 +16,41 @@ include { 'features/fetch-crl/rpms' + RPMS_CONFIG_SUFFIX };
 # Define fetch-crl version if not defined when adding RPMs
 variable FETCH_CRL_VERSION ?= '3.0';
 
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
 # fetch-crl configuration
-# ----------------------------------------------------------------------------
-include { 'components/sysconfig/config' };
-"/software/components/sysconfig/files/fetch-crl/CRLDIR" = SITE_DEF_CERTDIR;
-"/software/components/sysconfig/files/fetch-crl/FORCE_OVERWRITE" = if ( FETCH_CRL_FORCE_OVERWRITE ) {
-                                                                     'yes';
-                                                                   } else {
-                                                                     'no';
-                                                                   };
-"/software/components/sysconfig/files/fetch-crl/QUIET" = if ( FETCH_CRL_QUIET ) {
-                                                           'yes';
-                                                         } else {
-                                                           'no';
-                                                         };
-
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
+include { 'components/metaconfig/config' };
+# property daemons is not supported in older version of metaconfig but unfortunately
+# metaconfig doesn't publish its version until 15.12. Until the version is defined
+# use QUATTOR_RELEASE to determine the version.
+'/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/daemons' = {
```

Comment:
I tend to avoid `prefix`... a little bit more than a personal taste! One of the `prefix` side effect is that it's quite difficult to grep through the template library to identify the template modifying a particular path. Grepping `contents` or `daemons` will be useless and grepping the parent path only (the prefix) will sometime give too many matches to be efficient... But in this particular case, I agree that this `prefix` is probably configured only in this template so it should be ok.